lib/barcelona/network/auto_scaling_group.rb

```diff
@@ -11,7 +11,7 @@ def define_resource(json)
           j.DesiredCapacity desired_capacity
           j.Cooldown 0
           j.HealthCheckGracePeriod 0
-          j.MaxSize(desired_capacity + 1)
+          j.MaxSize(desired_capacity * 2 + 1)
```

Comment:
not batch size but `MinInstancesInService`. there is a limitation that is "MinInstancesInService must be less than the autoscaling group's MaxSize"